worlds/terraria/docs/setup_en.md

```diff
@@ -31,6 +31,7 @@ highly recommended to use utility mods and features to speed up gameplay, such a
     - (Can be used to break progression)
 - Reduced Grinding
 - Upgraded Research
+    - (Automatically researchers certain items so don't be afraid when they don't appear in your inventory)
```

Comment:
I find this wording to be awkward. I'd prefer it to be more straightforward and not mention being afraid.